libr/core/cmd_print.c

```diff
@@ -7492,7 +7492,7 @@ static int cmd_print(void *data, const char *input) {
 				int mode = input[2];
 				int wordsize = core->anal->config->bits / 8;
 				if (mode == '?') {
-					eprintf ("Usage: pxr[1248][*,jq] [length]\n");
+					R_LOG_ERROR ("Usage: pxr[1248][*,jq] [length]");
```

Comment:
same, dont turn the eprinf ("Usage with rlog, because thats not the right fix and hides the problem from lint.sh.

This may work with `r_core_cmd_help_match (core, help_msg_px, "pxr", false);
`